### DISCUSSION TITLE
Stellarium Alt-TAB Crashes

### DISCUSSION THREAD

**Discussion by FirChlis** [Stellarium/stellarium#3568] [General]
Hi there. I'm currently testing my new installation of Stellarium with EQMOD to drive an HEQ5 Pro mount, but I'm finding that Stellarium is crashing regularly, and it appears to be after an Alt-TAB between Stellarium and EQMOD (e.g. when doing the initial star alignment). I've tried this on two laptops, one running Windows 10 and one running Windows 11, with similar results.

Crash is reproducible: Start Stellarium; Connect Telescope; Alt-TAB 3-4 times between Stellarium and EQMOD; Stellarium crashes, leaving EQMOD still operable. Restarting Stellarium usually allows a reconnect to the telescope.

I've tried with the standard Stellarium, and also in MESA mode, with the same results.

Stellarium 23.3 release
EQMOD V200w
Ascom 662.4195
HEQ5 Pro Mount connected with Lynx Astro cable
Windows 10 Home

The last entry in the Stellarium log file (attached) is:

OleInvoke: Exception [scode:  -2147220480 wcode:  0  puArgErr:  0 ] GLE:  0
Could not get DoesRefraction for device:  "EQMOD.Telescope"

If it's of any use, Alt-TAB switching between Carte du Ciel and EQMOD work fine - but I prefer Stellarium, so I would like to get it working reliably.

If anyone has any suggestions...

Thank you.

Geoff

[log.txt](https://github.com/Stellarium/stellarium/files/13753405/log.txt)

**Comment by 10110111** [comment]
This sounds like a bug, which should've been reported as such rather than a "discussion". But note that it's most likely a duplicate of #2874.

**Reply by FirChlis** [reply]
Apologies - my first posting, I didn't see the bugs category. I'll see what I can do with alternative versions of Stellarium.

**Comment by FirChlis** [comment]
Early days yet, but uninstalling Stellarium 23.3 qt6 and replacing it with 23.3 qt5 appears to have got around the problem.

**Comment by zenmetsu** [comment]
I have an iOptron iEQ30 and am using their Commander software to control the mount via ascom/Stellarium.  

The problem described here was plaguing me as well, on Stellarium 23.4 Qt6.  I've switched to 23.4 built against Qt5 and the issue was resolved.
